tests/util.py

```diff
@@ -0,0 +1,21 @@
+# Standard Library
+import os
+
+
+class EnvManager(object):
```

Comment:
Is this file reqd. We have tests which are setting and unsetting env variables. We should reuse the same way.